test_install/CMakeLists.txt

```diff
@@ -33,29 +43,57 @@ if(GINKGO_BUILD_CUDA)
             ginkgo_switch_to_windows_static("CUDA")
         endif()
     endif()
-    add_executable(test_install_cuda test_install_cuda.cu)
+    configure_file(test_install.cpp test_install.cu COPYONLY)
+    add_executable(test_install_cuda test_install.cu)
```

Comment:
Does this place (and use) the file in the source or binary dir?

Reply:
This uses the one in the binary dir without issue. Possibly since there is no `test_install.cu` in the source dir? I was surprised about that as well.

Reply:
Just to be sure can we maybe use `CMAKE_CURRENT_BINARY_DIR` instead? otherwise this might be ambiguous if for some reason you still have the old file lying around.